shared/scss/components/_dropdown.scss

```diff
@@ -0,0 +1,18 @@
+// DROPDOWN Component
+// This is a generic dropdown
+// Whenever .dropdown is hovered .dropdown__content appears 
```

Comment:
For the comment in the beginning please use the better structure like:
```
/*=========================================================
*********   Generic DropDown Component
*******     Whenever .dropdown is hovered .dropdown__content appears
*/
```

but, question, this component is useful for other project?! if not move it to your project.

Reply:
I think it can be useful because it is a common pattern when an element is hovered another element that contains a content related to it appears